target/linux/ath79/image/generic.mk

```diff
@@ -899,10 +899,28 @@ define Device/netgear_wndr3800
   NETGEAR_BOARD_ID := WNDR3800
   NETGEAR_HW_ID := 29763654+16+128
   IMAGE_SIZE := 15872k
-  SUPPORTED_DEVICES += wndr3700
+  SUPPORTED_DEVICES += wndr3800
```

Comment:
I don't think that's correct. Despite, if it's valid, it should go into a separate commit.

Reply:
Fair. I'll remove that.